client/test/App.spec.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import App, { Counter } from '../src/App';
+
+describe('App', () => {
+  test('snapshot renders', () => {
+    const component = renderer.create(<App />);
```

Comment:
Cool. A snapshot test.  Is this working, I'd love to see it working.

Reply:
@rickboot Yes. You can check this out in your __snapshots__ folder when you test this. Pretty cool!